samples/android/GleanCrashExample/LICENSE.md

```diff
@@ -0,0 +1,363 @@
+
+Mozilla Public License, version 2.0
```

Comment:
Would it be better to license the example in the public domain?  Developers may be disinclined to use it as the basis for production work otherwise...  (I know the MPL should be fine... it's more about optics...)

Reply:
Good question! I just followed suit on what the sample application does in regards to licensing, the only difference is I included the entire license file rather than just the header at the top of the `MainActivity.kt`